ansible/vars/dev-vagrant.yml

```diff
@@ -30,3 +30,7 @@ site_user_email_confirmation_enabled: false
 
 # Algolia
 site_algolia_search_prefix: 'dev_'
+
+# Notification mailer
+notification_mailer_address: "notifications@librecores.org"
+notification_sender_name: "LibreCores"
```

Comment:
- Set the same values also for staging and production. (Use stage.librecores.org for staging emails.)
- Use "LibreCores Notifications" as name.